pages/LoginPage.js

```diff
@@ -0,0 +1,22 @@
+import { getAuthData } from '../utils';
+
+export class LogIn {
```

Comment:
Lets keep in mind, its the page
```suggestion
export class LoginPage {
```